### PR TITLE
added colors.py [color_scale, rdm_colormap] and test_colors.py [passed]

### PR DESCRIPTION
**Pull request by iancharest**
unit tests have passed on my machine. please review.

### CODE REVIEW

File: tests/test_colors.py [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+test_colors
+Test for colors 
+@author: iancharest
+"""
+
+import unittest
+from pyrsa.vis.colors import color_scale
+from pyrsa.vis.colors import rdm_colormap
+
+class ColorTests(unittest.TestCase):
```
Comment (JasperVanDenBosch) [inline tests/test_colors.py:14]: Perhaps you can add a quick test that actually tests the colors, with a dummy rdm for example, checking that cell 1 is brighter than cell2 or whatever the goal of the colormap is. Currently if the function just returns a bunch of nans, you'd never know

File: tests/test_colors.py [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+    def test_rdm_colormap(self):
+        n_cols = 10
+        cols = rdm_colormap(n_cols)
+        n_cols_returned, n_rgb = cols.colors.shape
+        self.assertEqual(n_cols_returned, n_cols)
+        self.assertEqual(n_rgb, 3)
+
+if __name__ == '__main__':
+    unittest.main()
```
Comment (JasperVanDenBosch) [inline tests/test_colors.py]: this part is not necessary I think, if you want to run one test case you can call:

 `python setup.py test -s tests.test_colors.ColorTests`

**Review by JasperVanDenBosch — CHANGES_REQUESTED** [review]

**Comment by JasperVanDenBosch** [comment]
resolves #19

File: tests/test_colors.py (hunk 1 above)
Comment (iancharest) [reply tests/test_colors.py:14]: working on it.

File: tests/test_colors.py (hunk 2 above)
Comment (iancharest) [reply tests/test_colors.py]: i've added a test case. the last colour returned from rdm_colormap should by design be yellow.

**Review by JasperVanDenBosch — APPROVED** [review]

**Review by doerlbh — APPROVED** [review]

**Pull request merged**